client/lib/olark/index.js

```diff
@@ -151,6 +151,12 @@ const olark = {
 		}
 	},
 
+	fetchUserEligibility() {
+		this.getOlarkConfiguration()
+			.then( ( configuration ) => this.configureOlark( configuration ) )
```

Comment:
Calling `configureOlark` more than once in the page lifetime will cause unpredictable issues because we'll end up listening to for [these](https://github.com/Automattic/wp-calypso/blob/fix/1827-update-contact-option/client/lib/olark/index.js#L118) events more than once.